packages/react-core/src/components/MultipleFileUpload/MultipleFileUploadStatusItem.tsx

```diff
@@ -50,6 +50,8 @@ export interface MultipleFileUploadStatusItemProps extends React.HTMLProps<HTMLL
   progressAriaLabel?: string;
   /** Associates the progress bar with it's label for accessibility purposes. Required when title not used */
   progressAriaLabelledBy?: string;
+  /** Adds an accessible aria live region to indicate updates to the progress bar. */
```

Comment:
Since the live region gets rendered regardless if this is passed, the description could be more along the lines of:

```suggestion
  /** Modifies the text announced by assistive technologies when the progress bar updates. */
```

Or essentially just something that will convey the prop will modify the default live region behavior.